src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -496,10 +496,23 @@ JSON example:
 
 ### Fieldmap data
 
-Data acquired to correct for B0 inhomogeneities can come in different forms. The
-current version of this standard considers four different scenarios. Please note
-that in all cases fieldmap data can be linked to a specific scan(s) it was
-acquired for by filling the IntendedFor field in the corresponding JSON file.
+Both B0 (static magnetic field strength pattern), B1+ (transmit field pattern), and 
+B1- (receive field pattern; not yet supported) maps can be useful in post-processing
+both raw functional and anatomical data.
+
+B0 maps are primarily used to correct for spatial distortions in functional
+data acquired with EPI sequences.
+
+B1+ and B1- maps are mostly used in anatomical imaging, especially when applying
+quantitative MRI (qMRI or hMRI) techniques.
```

Comment:
Minor concern, `quantitative MRI (qMRI or hMRI)` still confuses me :)

Reply:
Yeah good point. Let's just keep it `qmri`.